### DISCUSSION TITLE
To produce HTML, maybe we could use pandoc?

### DISCUSSION THREAD

**Discussion by bligneri** [fenjalien/obsidian-typst#66] [Ideas]
It seems like pandoc has a converter to and from typst.

https://pandoc.org/

In particular:
https://pandoc.org/typst-property-output.html

of course it may be a bit of a challenge to support all platforms but it may be a nice way to integrate with Obsidian?

**Comment by coycatrett** [comment]
Has this idea been considered further in the past 10 months?
